build/SharedFx.targets

```diff
@@ -0,0 +1,142 @@
+<Project>
+  <Import Project="common.props" />
+  <Import Project="SharedFx.props" />
+
+  <Target Name="BuildMetapackage" DependsOnTargets="ResolveRepoInfo">
+    <!-- Clear working directory -->
+    <RemoveDir Directories="$(_WorkRoot)" />
+
+    <!-- Move to working dir -->
```

Comment:
Comment doesn't make sense here. What's moving?

Reply:
Moved comment to the copy task below. It used to make sense when the properties and itemgroups were defined for the copy task.